<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="uploadFile">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>1</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="mainTextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>491</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="wordSearchBar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search_wordButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="translationBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type translation here if unknown:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translation GERMAN -&gt; ENGLISH</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="enterTranslationBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_new_word_to_dictionaryButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadGerman">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload GERMAN dictionary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadEnglish">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload ENGLISH translation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_three_upload_complete">
    <property name="geometry">
     <rect>
      <x>562</x>
      <y>0</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Click here when all files are loaded</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="englishBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="germanBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>English dictionary</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>German dictionary</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveNewVocabFile">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save new vocabulary file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGerman_English_Reading">
    <property name="title">
     <string>German-&gt;English Reading </string>
    </property>
   </widget>
   <addaction name="menuGerman_English_Reading"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
